src/SFA.DAS.ApprenticeAccounts/Application/Commands/CreateMyApprenticeCommand/MyApprenticeshipCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticeAccounts.Application.Commands.CreateMyApprenticeCommand;
+
+public interface IMyApprenticeshipCommand
```

Comment:
eh it was just a way to enforce same fields in both request and command, but it's superfluous because mappings already do that, as well as coverage. Taking out